plugin/trino-delta-lake/pom.xml

```diff
@@ -17,14 +17,13 @@
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
 
         <!--
-          Project's default for air.test.parallel is 'methods'. By design, 'classes' makes TestNG run tests from one class in a single thread.
```

Comment:
Can we update this in airbase too? This sounds like a much safer default.

Reply:
This setting is to be defined in the concrete project extending from airbase.

https://github.com/airlift/airbase/blob/master/airbase/pom.xml#L102-L105

I'm setting `instances` to be the values in `trino-root/pom.xml` file for `air.test.parallel`.

A potential downside can be long tail single-threaded execution of a single long test class.

Reply:
Generally `methods` is a better default since that allows concurrency even if you have a single test class that's running. In some modules in Trino however we're faced with a resource limitation so `instances` works (until Surefire is fixed).

Reply:
Yes, i think we discussed this offline and didn't follow-up here.
Long tail is a very likely problem with tests executions, that's why we still prefer `methods` by default.